lib/chai/config.js

```diff
@@ -50,6 +50,22 @@ module.exports = {
    * @api public
    */
 
-  truncateThreshold: 40
+  truncateThreshold: 40,
 
+  /**
+   * ### config.useProxy
+   *
+   * User configurable property, defines if chai will use a Proxy to throw
+   * an error when a non-existent property is read, which protects users
+   * from typos when using property-based assertions.
+   *
+   * Set it to false if you want to disable this feature.
```

Comment:
I just had another thought on this: We should mention that this feature is automatically disabled regardless of this config value in environments that don't support proxies.